src/components/material-grid-related/MaterialGridRelated.tsx

```diff
@@ -19,26 +39,123 @@ const MaterialGridRelated: React.FC<MaterialGridRelatedProps> = ({ work }) => {
   const title = t("materialGridRelatedTitleText");
 
   const pid = getWorkPid(work);
-  const { data } = useWorkRecommendationsQuery({
-    pid: pid,
-    limit: 8
-  });
 
-  if (!data?.recommend.result) {
+  const { creators } = work.manifestations.bestRepresentation;
+  const { series } = work;
+  const seriesObject = first(series);
+  const flattenedCreators = flattenCreators(creators);
+  const creatorCqlString = prepareCreatorCql(flattenedCreators);
+
+  const { data: recommendationData, isLoading: recommendationLoading } =
+    useWorkRecommendationsQuery({
+      pid,
+      limit: 8
+    });
+
+  const { data: creatorData, isLoading: creatorLoading } =
+    useComplexSearchWithPaginationQuery({
+      cql: creatorCqlString,
+      limit: 8,
+      offset: 0,
+      filters: {}
+    });
+
+  const { data: seriesData, isLoading: seriesLoading } =
+    useComplexSearchWithPaginationQuery({
+      cql: `term.series='${seriesObject?.title}'`,
+      limit: 8,
+      offset: 0,
+      filters: {}
+    });
+
+  const [filter, setFilter] =
+    React.useState<MaterialGridFilterType>("recommendation");
+
+  const allQueriesLoaded =
+    !recommendationLoading && !creatorLoading && !seriesLoading;
+
+  const recommendationMaterials =
+    extractRecommendationMaterials(recommendationData);
+  const seriesMaterials = extractSeriesMaterials(seriesData);
+  const authorMaterials = extractAuthorMaterials(creatorData);
+
+  const options = React.useMemo<MaterialGridRelatedOption[]>(() => {
+    if (!allQueriesLoaded) return [];
+
+    const opts: MaterialGridRelatedOption[] = [];
+
+    if (recommendationMaterials.length) {
+      opts.push({
+        label: t("materialGridRelatedRecommendationsDataLabelText"),
+        value: "recommendation",
+        materials: recommendationMaterials
+      });
+    }
+
+    if (seriesMaterials.length) {
+      opts.push({
+        label: t("materialGridRelatedSeriesDataLabelText"),
+        value: "series",
+        materials: seriesMaterials
+      });
+    }
+
+    if (authorMaterials.length) {
+      opts.push({
+        label: t("materialGridRelatedAuthorDataLabelText"),
+        value: "author",
+        materials: authorMaterials
+      });
+    }
+
+    return opts;
+  }, [
+    allQueriesLoaded,
+    recommendationMaterials,
+    seriesMaterials,
+    authorMaterials,
+    t
+  ]);
+
+  React.useEffect(() => {
```

Comment:
Not a huge deal, but I noticed that we usually deconstruct useEffect from the React import elsewhere. It might be good to stick with the same pattern for consistency.